### ISSUE TITLE
New metric: F1

### ISSUE THREAD

**Issue by lazappi** [theislab/atlas-feature-selection-benchmark#17]
Thanks for taking the time to suggest a new metric!

## Description

> _Please briefly describe the suggested metric: What is it? What does it measure? Why would it be a good fit for the project?_

The F1 score is a commonly used evaluation metric that measures classification performance as the harmonic mean of precision and recall. 

## Links

> _Links to information about the metric_

- **Description:** https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html
- **Code:** https://github.com/scikit-learn/scikit-learn
- **Package:** https://pypi.org/project/sklearn/

## Anything else?

> _Anything else you think is important about the metric_

- Need to decide what kind of class averaging to use
